RadixWallet/Features/SecurityCenterFeature/SecurityFactors/FactorSourcesList+View.swift

```diff
@@ -0,0 +1,206 @@
+// MARK: - FactorSourcesList.View
+extension FactorSourcesList {
+	@MainActor
+	struct View: SwiftUI.View {
+		let store: StoreOf<FactorSourcesList>
+
+		var body: some SwiftUI.View {
+			WithViewStore(store, observe: { $0 }) { viewStore in
+				ScrollView {
+					VStack(spacing: .large3) {
+						header(viewStore.kind.details)
+
+						if let main = viewStore.main {
+							section(text: L10n.FactorSources.List.default, rows: [main])
+								.padding(.top, .medium3)
+
+							if !viewStore.others.isEmpty {
+								section(text: L10n.FactorSources.List.others, rows: viewStore.others)
+							}
+						} else {
+							section(text: nil, rows: viewStore.others)
+						}
+
+						Button(viewStore.addTitle) {
+							store.send(.view(.addButtonTapped))
+						}
+						.buttonStyle(.secondaryRectangular)
+
+						let infoContent = viewStore.kind.infoLinkContent
+						InfoButton(infoContent.item, label: infoContent.title)
+					}
+					.padding(.medium3)
+					.padding(.bottom, .medium2)
+				}
+				.background(.app.gray5)
+				.radixToolbar(title: viewStore.kind.title)
+				.task {
+					store.send(.view(.task))
+				}
+			}
+			.destinations(with: store)
+		}
+
+		private func header(_ text: String?) -> some SwiftUI.View {
+			Text(text)
+				.textStyle(.body1Header)
+				.foregroundStyle(.app.gray2)
+				.flushedLeft
+		}
+
+		private func section(text: String?, rows: [State.Row]) -> some SwiftUI.View {
+			VStack(spacing: .small1) {
+				header(text)
+
+				ForEachStatic(rows) { row in
+					card(row)
+				}
+			}
+		}
+
+		private func card(_ row: State.Row) -> some SwiftUI.View {
+			FactorSourceCard(
+				kind: .instance(factorSource: row.integrity.factorSource, kind: .extended(linkedEntities: row.linkedEntities)),
+				mode: .display,
+				messages: row.messages
+			) { action in
+				switch action {
+				case .removeTapped:
+					break
+				case .messageTapped:
+					store.send(.view(.rowMessageTapped(row)))
+				}
+			}
+			.onTapGesture {
+				store.send(.view(.rowTapped(row)))
+			}
+		}
+	}
+}
+
+private extension FactorSourcesList.State {
+	var addTitle: String {
+		switch kind {
+		case .device:
+			L10n.FactorSources.List.deviceAdd
+		case .ledgerHqHardwareWallet:
+			L10n.FactorSources.List.ledgerAdd
+		case .offDeviceMnemonic:
+			L10n.FactorSources.List.passphraseAdd
+		case .arculusCard:
+			L10n.FactorSources.List.arculusCardAdd
+		case .password:
+			L10n.FactorSources.List.passwordAdd
+		case .trustedContact, .securityQuestions:
+			fatalError("Not implemented")
+		}
+	}
+
+	var main: Row? {
+		rows.first(where: \.integrity.isExplicitMain)
+	}
+
+	var others: [Row] {
```

Comment:
`	var others: [Row] {` should probably be in Sargon?

Reply:
this is just a differentiation for whether we should show a header and main device, or if they get all mixed up together. Being plainly UI, I don't think it is worth moving to Sargon.